### ISSUE TITLE
Feature Request: "Create New Window With Title"

### ISSUE THREAD

**Issue by jeremymeyers** [tpamula/webextension-window-titler#54]
Sometimes when I'm moving tabs around it would be useful to have the ability to create a new named window from the extension rather than having to create a new window and then name it.  Is this doable?

**Comment by m-ar-c** [comment]
Yeah, like a submenu with all the preset titles. You click on one and it would open the window already named. And you know what would be extra useful with that ? The possibility to associate preset titles with bookmark folders ! That way you click on some project name and an already labeled window with all the pages you need to work on are opened ! Neat.
